base/div.jl

```diff
@@ -80,6 +80,16 @@ See also: [`div`](@ref)
 julia> fld(7.3,5.5)
 1.0
 ```
+Because of floating-point rounding `fld(x,y)` can lead to an *alleged* violation of the fld(x,y) == x/y condition:
```

Comment:
Might be nice to have backticks around the `fld(x,y) == x/y`

Reply:
Good idea, fixed that together with a mistake that made doctests fail.